File: Java/SortList.java
// SortList.java
import java.util.Arrays;

public class SortList {
    public static void bubbleSort(int[] arr) {
        int n = arr.length;
        for (int i = 0; i < n - 1; i++) {
            for (int j = 0; j < n - i - 1; j++) {
                if (arr[j] > arr[j + 1]) {
                    // Swap elements
                    int temp = arr[j];
                    arr[j] = arr[j + 1];
                    arr[j + 1] = temp;
                }
            }
        }
    }

    public static void main(String[] args) {
        int[] numbers = {5, 3, 8, 1, 9, 2};
        int[] numbersCopy = numbers.clone();
        
        System.out.println("Original array: " + Arrays.toString(numbers));
        
        Arrays.sort(numbers);
        System.out.println("Sorted using Arrays.sort(): " + Arrays.toString(numbers));
        
        bubbleSort(numbersCopy);
        System.out.println("Sorted using bubble sort: " + Arrays.toString(numbersCopy));
    }
}

File: Python/sort_list.py
# sort_list.py

def sort_numbers(numbers):
    """Sort a list of numbers in ascending order."""
    return sorted(numbers)

def bubble_sort(numbers):
    """Sort using bubble sort algorithm."""
    n = len(numbers)
    arr = numbers.copy()
    for i in range(n):
        for j in range(0, n - i - 1):
            if arr[j] > arr[j + 1]:
                arr[j], arr[j + 1] = arr[j + 1], arr[j]
    return arr

if __name__ == "__main__":
    test_list = [5, 3, 8, 1, 9, 2]
    sorted_list = sort_numbers(test_list)
    bubble_sorted = bubble_sort(test_list)
    print(f"Original: {test_list}")
    print(f"Sorted (built-in): {sorted_list}")
    print(f"Sorted (bubble sort): {bubble_sorted}")
